src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.scss

```diff
@@ -130,4 +130,48 @@
             width: 100%;
         }
     }
+
+    .question-card-header {
+        width: 100%;
+        padding: 15px 5px 5px 5px;
+
+        .row {
+            width: 100%;
+        }
+
+        .question-collapse {
+            margin-top: -15px;
+            width: 3.25rem;
+        }
+    }
+
+    .card-body {
+        .form-group {
+            display: flex;
+            align-items: center;
+            margin: 0 5px 15px;
+        }
+
+        .form-control:disabled {
+            background-color: #e9ecef;
```

Comment:
hard coded colors are not allowed. see theming-guide

Reply:
Changed this to `var(--gray-200)`.